benefits/static/css/styles.css

```diff
@@ -558,6 +566,87 @@ h1 + .media-list, /* A .media-list immediately following the h1: Enrollment Succ
   list-style-type: disc;
 }
 
+/* Cards */
+
+:root {
+  --card-title-font-size: var(--font-size-18px);
+  --card-x-padding: calc(10rem / 16);
+  --card-body-x-padding: 0;
+  --card-body-y-padding: 0.75rem;
+  --card-border-width: var(--font-size-12px);
+}
+
+@media (min-width: 992px) {
+  :root {
+    --card-title-font-size: var(--font-size-20px);
+    --card-x-padding: calc(35rem / 16);
+    --card-body-x-padding: calc(40rem / 16);
+    --card-body-y-padding: 0;
+    --card-border-width: var(--font-size-20px);
+  }
+}
+
+.card {
+  height: 75%;
+  border-top: solid var(--card-border-width) var(--primary-color);
+  border-radius: 8px;
+  transition: 0.3s;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  padding: var(--card-body-y-padding) var(--card-x-padding);
+}
+
+.card:hover {
+  border-top: solid var(--card-border-width) var(--hover-color);
+}
+
+.card .card-title {
+  font-size: var(--card-title-font-size);
```

Comment:
The `.card-title` line-height is 24px/120% on desktop and 21.6px on mobile on Figma. The mobile one doesn't matter as much, but it matters a lot for desktop b/c the agency names go into 2 lines.

Reply:
Done in 0c29a5f12211c976c222f2bbca8db6ed69c9d4b5